int64 joint_id
---
string reason
